pkg/docker/config/config.go

```diff
@@ -51,6 +51,14 @@ func SetAuthentication(sys *types.SystemContext, registry, username, password st
 	})
 }
 
+func getAdditionalAuthFiles(sys *types.SystemContext) []string {
```

Comment:
Nit: Golang frowns on beginning commands with get. Just call it additionalAuthFiles

Reply:
Most functions in this file start with `{get,set}` but I apply the change.